server/src/modules/models/video/service.js

```diff
@@ -75,10 +75,7 @@ const updateViewCount = async (id) => {
 
 const deleteById = async (id) => {
   try {
-    const deleted = await Video.deleteOne({
-      _id: new ObjectId(id),
-    });
-    return deleted;
+    return await Video.deleteById(id);
```

Comment:
use the similar pattern/way of `Video.getObjectById,` so that we can remove this method.

Reply:
Nice suggestion.